woocommerce/woocommerce-android
# Add toolbar for ShippingCustomsFragment

Part of #10570 

### Description

This PR introduces individual toolbars for ShippingCustomsFragment screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Order detail screen of an Order that has a shipping label (PbTz5e-11v-p2) (Order list -> Order detail -> Create a shipping label CTA -> Ship from -> Ship to -> Package details -> Customs) screen and ensure the toolbar and its action work as expected on both phone and tablet.

### Images/gif

<img width="241" alt="Screenshot 2024-02-12 at 6 55 35 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/ce7496c5-beb4-4b0a-9697-4d6316a4a23e">
